Scripts/Runtime/Entities/Tasks/Data/AbstractTaskWorkConfig.cs

```diff
@@ -76,6 +78,17 @@ private void AddDataWrapper(AbstractVDWrapper dataWrapper)
 #endif
         }
 
+        protected void InternalRequireResultsDestinationLookup<TKey, TInstance>(VirtualData<TKey, TInstance> data, bool isAsync)
```

Comment:
Should this be a debug method with a conditional since it's only used for collections checks?

Reply:
Yes if its going to stay that way. Right now I'm not 100% sure. I might want to use it a wrapper still. 

Added TODO and reference back to here.